app-ios/Sources/CommonComponents/LoadingView.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+public struct LoadingView: View {
+
+    public init() {}
+
+    public var body: some View {
+        ZStack {
+            Color.black
```

Comment:
Please use background color defined in Theme

Reply:
I fixed at https://github.com/DroidKaigi/conference-app-2022/pull/656/commits/263759bafe64ae4a91351949769ab98797462cbd !